libs/application/template-api-modules/src/lib/modules/shared/shared.service.ts

```diff
@@ -153,10 +150,13 @@ export class SharedTemplateApiService {
   async sendEmailWithAttachment(
     templateGenerator: AttachmentEmailTemplateGenerator,
     application: Application,
-    fileContent: string,
+    fileContent: Parameters<AttachmentEmailTemplateGenerator>[1],
     recipientEmail: string,
     locale = 'is',
   ) {
+    this.logger.debug('Sending email with attachment', {
+      applicationId: application.id,
+    })
```

Comment:
Should this be here?

Reply:
I think for debugging purposes this is good to have, so I think it was included on purpose

Reply:
In our live environment, debug logs are only logged when enabled specifically for an app. I.e. this has "no impact" unless we need the debug logs, in which case we can turn it on.

This was indeed intentional.